shm/Game.cpp

```diff
@@ -260,6 +271,8 @@ void Game::makeAction(Action choice) {
 
 void Game::startGame() {
     while (true) {
+        currentPosition_ = map_->getCurrentPosition();
```

Comment:
There is no need for it to be called on every loop. Updating it after travel would be better